pino-elasticsearch.js

```diff
@@ -112,7 +109,10 @@ if (require.main === module) {
       version: 'v',
       help: 'h',
       host: 'H',
-      port: 'p'
+      port: 'p',
+      index: 'i',
+      size: 's',
+      loglevel: 'l'
```

Comment:
Any suggestion "--elasticsearch-trace-level" and "--bulk-size". I think I did change usage.txt  ... anyhow it will change again :)